src/routes/load/components/OwnerList/index.tsx

```diff
@@ -100,7 +100,7 @@ const OwnerListComponent = (props) => {
                     placeholder="Owner Name*"
                     text="Owner Name"
                     type="text"
-                    validate={composeValidators(required, minMaxLength(1, 50))}
+                    validate={composeValidators(required, minMaxLength(1, 50), validAddressBookName)}
```

Comment:
This will force to set a name to all owners when loading a Safe

Reply:
oops! thanks, just removed it.